frontend/src/utils/Firebase.ts

```diff
@@ -0,0 +1,14 @@
+import * as firebase from "firebase/app";
```

Comment:
Minor nit but can u rename this to firebase.ts

Reply:
renamed file to firebase.ts